framework/src/governance/lib/data-lake-catalog.ts

```diff
@@ -67,12 +67,17 @@ export class DataLakeCatalog extends TrackedConstruct {
     const extractedBronzeBucketName = this.extractBucketName(props.dataLakeStorage.bronzeBucket);
     const extractedSilverBucketName = this.extractBucketName(props.dataLakeStorage.silverBucket);
     const extractedGoldBucketName = this.extractBucketName(props.dataLakeStorage.goldBucket);
-    const locationPrefix = props.databaseName || '/';
+    const bronzeDatabaseName = props.databaseName ? `${extractedBronzeBucketName}_${props.databaseName}` : extractedBronzeBucketName;
+    const silverDatabaseName = props.databaseName ? `${extractedSilverBucketName}_${props.databaseName}` : extractedSilverBucketName;
+    const goldDatabaseName = props.databaseName ? `${extractedGoldBucketName}_${props.databaseName}` : extractedGoldBucketName;
+    const bronzeLocationPrefix = props.databaseName || bronzeDatabaseName;
```

Comment:
`const bronzeLocationPrefix = props.databaseName || extractedBronzeBucketName`

Should be like this because we never go into the first condition of `bronzeDatabaseName`

Reply:
Fixed